Doc/library/tempfile.rst

```diff
@@ -162,7 +162,9 @@ The module defines the following user-callable items:
    by child processes.
 
    Unlike :func:`TemporaryFile`, the user of :func:`mkstemp` is responsible
-   for deleting the temporary file when done with it.
+   for deleting the temporary file when done with it.If a large number of file
+   descriptors are created, your program may run into a per-process limit and crash.
```

Comment:
I think we shouldn't say "and crash" because it implies a hard crash, but on OS X an OSError is raised instead, and I'm fairly sure on Linux as well; on Windows I'm not quite sure but from what I was able to find, Windows has a much higher handle limit, so this warning more or less isn't needed for Windows.

Now that I think of it, it may be good to add something about OSError being raised (because users might want to catch it if they want to keep many FDs around potentially but want to do a graceful exit if the limit is hit), but then we need to test it on a Unix system. Srinivas, if you have access to a unix system, can you test that? If not, we can ask somebody on issue tracker or on the IRC channel.

I tested like so, after checking `ulimit -n`, -- `for x in range(300): tempfile.mkstemp()` # OSError.

Reply:
Mine is OSX, and I do not have access to Unix.

Reply:
I've confirmed (on the issue page) that OSError happens in Linux as well.